examples/nmt_transformer/french_translation/README.md

```diff
@@ -0,0 +1,58 @@
+# English-to-French translation using Fairseq Transformer model
```

Comment:
This is already covered in the main README of this example?

Reply:
Removed french_translation and german_translation folders.

Reply:
Done